src/Stratis.Bitcoin.Features.Wallet/Wallet.cs

```diff
@@ -98,11 +98,15 @@ public IEnumerable<HdAccount> GetAccountsByCoinType(CoinType coinType, Func<HdAc
         /// </summary>
         /// <param name="accountName">The name of the account to retrieve.</param>
         /// <param name="coinType">The type of the coin this account is for.</param>
+        /// <param name="missingThrowsError">Set to <c>true</c> to throw an error if the account is missing and <c>false</c> to return <c>null</c>.</param>
         /// <returns>The requested account.</returns>
-        public HdAccount GetAccountByCoinType(string accountName, CoinType coinType)
+        public HdAccount GetAccountByCoinType(string accountName, CoinType coinType, bool missingThrowsError = true)
```

Comment:
not sure if `bool missingThrowsError` is a good approach. Probably better would be to go with single way to handle missing account and enforce users of this method to handle the result as they please

Reply:
Done.